.github/workflows/java_api.yml

```diff
@@ -45,28 +47,52 @@ jobs:
           export ONNXRUNTIME_DIR=${PWD}
           export LD_LIBRARY_PATH=$ONNXRUNTIME_DIR/lib:$LD_LIBRARY_PATH
           popd
-      - name: Install opencv
+      - name: Install OpenCV
         run: |
           sudo apt-get install libopencv-dev
-      - name: Build java class
+      - name: Install ant
+        run: |
+          sudo apt-get install ant
+          export ANT_HOME=/usr/share/ant
+      - name: Build Java class
         run: |
           pushd csrc/mmdeploy/apis/java
           javac mmdeploy/*.java
           popd
-          pushd demo/java
-          javac -classpath ../../csrc/mmdeploy/apis/java/ Utils.java
-          popd
-      - name: Install mmdeploy with onnxruntime backend and java api
+      - name: Install mmdeploy with onnxruntime backend and Java API
         run: |
           mkdir -p build && pushd build
-          export LD_LIBRARY_PATH=/home/runner/work/mmdeploy/mmdeploy/ncnn/install/lib/:$LD_LIBRARY_PATH
+          export LD_LIBRARY_PATH=$GITHUB_WORKSPACE/mmdeploy/mmdeploy/ncnn/install/lib/:$LD_LIBRARY_PATH
           cmake -DMMDEPLOY_BUILD_SDK=ON -DMMDEPLOY_BUILD_SDK_JAVA_API=ON -DMMDEPLOY_TARGET_BACKENDS=ort -DMMDEPLOY_CODEBASES=all -DONNXRUNTIME_DIR=~/work/mmdeploy/mmdeploy/onnxruntime-linux-x64-1.8.1  ..
           make install
+          echo $PWD
+          find $GITHUB_WORKSPACE/mmdeploy/mmdeploy/build/lib/libmmdeploy_java.so
           popd
-      - name: Test javademo
+      - name: Prepare OpenCV Java and test Java demo
         run: |
-          export JAVA_HOME=${PWD}/jdk-18
-          export PATH=${JAVA_HOME}/bin:${PATH}
-          export LD_LIBRARY_PATH=/build/lib:${LD_LIBRARY_PATH}
-          java --version
-          python .github/scripts/test_java_demo.py
+          echo ${JAVA_HOME}
+          export OPENCV_VERSION=4.7.0
+          export JAVA_AWT_INCLUDE_PATH=${JAVA_HOME}
+          export JAVA_AWT_LIBRARY=${JAVA_HOME}
+          export JAVA_INCLUDE_PATH=${JAVA_HOME}/include
+          export JAVA_INCLUDE_PATH2=${JAVA_HOME}/include/darwin
+          export JAVA_JVM_LIBRARY=${JAVA_HOME}
+          pushd $GITHUB_WORKSPACE
+          mkdir opencv && pushd opencv
+          mkdir -p build && pushd build
+          mkdir -p bin && pushd bin
+          wget https://media.githubusercontent.com/media/hanrui1sensetime/mmdeploy-javaapi-testdata/master/opencv-470.jar
+          popd
+          wget https://media.githubusercontent.com/media/hanrui1sensetime/mmdeploy-javaapi-testdata/master/opencv470-lib.tar
+          tar -xf opencv470-lib.tar -C ./
+          popd
+          export OPENCV_DIR=${PWD}
+          echo $OPENCV_DIR
+          popd && popd
+          find $GITHUB_WORKSPACE/mmdeploy/mmdeploy/build/lib/libmmdeploy_java.so
+          pushd demo/java
+          sudo apt-get install libgstreamer1.0-0 libgstreamer-plugins-base1.0-0 libavcodec-dev libavformat-dev libswscale-dev
```

Comment:
May put `apt-get` to `Install OpenCV`

Reply:
> May put `apt-get` to `Install OpenCV`

Seems not because of missing opencv.

Reply:
What I mean is `sudo apt-get install ffmpeg libgstreamer1.0-0 libgstreamer-plugins-base1.0-0 libavcodec-dev libavformat-dev libswscale-dev` can be put under the step "Install OpenCV".
The motivation is trying to put the opencv related dependencies under the same step